src/main/java/com/github/tomakehurst/wiremock/common/UniqueFilenameGenerator.java

```diff
@@ -26,20 +26,13 @@ public static String generate(String url, String prefix, String id) {
 
   public static String generate(String url, String prefix, String id, String extension) {
     String pathPart = Urls.urlToPathParts(URI.create(url));
-    pathPart = pathPart.equals("") ? "(root)" : sanitise(pathPart);
+    pathPart = pathPart.isEmpty() ? "(root)" : sanitise(pathPart);
```

Comment:
Ideally should be `isBlank()` so that we handle more edge cases, but out of the scope for the PR title anyway